ElementX/Sources/AccessibilityTests/AccessibilityTestsAppCoordinator.swift

```diff
@@ -0,0 +1,152 @@
+//
+// Copyright 2025 New Vector Ltd.
+//
+// SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+// Please see LICENSE files in the repository root for full details.
+//
+
+import Combine
+import MatrixRustSDK
+import SwiftUI
+import UIKit
+
+class AccessibilityTestsAppCoordinator: AppCoordinatorProtocol {
+    var windowManager: any SecureWindowManagerProtocol
+    
+    func handleDeepLink(_ url: URL, isExternalURL: Bool) -> Bool {
+        fatalError("Not implemented")
+    }
+    
+    func handlePotentialPhishingAttempt(url: URL, openURLAction: @escaping (URL) -> Void) -> Bool {
+        fatalError("Not implemented")
+    }
+    
+    func handleUserActivity(_ userActivity: NSUserActivity) {
+        fatalError("Not implemented")
+    }
+    
+    private let previewsWrapper: PreviewsWrapper
+    
+    private var cancellables = Set<AnyCancellable>()
+    
+    init(appDelegate: AppDelegate) {
+        windowManager = WindowManager(appDelegate: appDelegate)
+        // disabling View animations
+        UIView.setAnimationsEnabled(false)
+        
+        MXLog.configure(currentTarget: "accessibility-tests")
+        
+        ServiceLocator.shared.register(userIndicatorController: UserIndicatorController())
+        
+        AppSettings.configureWithSuiteName("io.element.elementx.accessibilitytests")
+        AppSettings.resetAllSettings()
+        ServiceLocator.shared.register(appSettings: AppSettings())
+        ServiceLocator.shared.register(bugReportService: BugReportServiceMock(.init()))
+        
+        let analyticsClient = AnalyticsClientMock()
+        analyticsClient.isRunning = false
+        ServiceLocator.shared.register(analytics: AnalyticsService(client: analyticsClient,
+                                                                   appSettings: ServiceLocator.shared.settings))
+        
+        guard let name = ProcessInfo.accessibilityViewID,
+              let previewType = TestablePreviewsDictionary.dictionary[name]
+        else { fatalError("Unable to launch with unknown screen.")
+        }
```

Comment:
Probably should either be on 1 line or 3 :)

```suggestion
        else { fatalError("Unable to launch with unknown screen.") }
```